Lib/subprocess.py

```diff
@@ -951,6 +942,19 @@ def _make_inheritable(self, handle):
             return Handle(h)
 
 
+        def _filter_handle_list(self, handle_list):
+            """Filter out console handles that can't be used
+            in lpAttributeList["handle_list"] and make sure the list
+            isn't empty"""
```

Comment:
Please document also that it removes duplicate handles. (Move the comment into the docstring.)